docs/docusaurus.config.ts

```diff
@@ -167,12 +167,16 @@ const config: Config = {
       style: "dark",
       links: [
         {
-          title: "Docs",
+          title: "Learning",
           items: [
             {
               label: "Documentation",
               to: "/quickstart",
             },
+            {
```

Comment:
@morganmcg1 where does this show up?

Reply:
in the docs footer, see the Before and After screenshots